### PR TITLE
Adding Test Cases for TODOIST/Sections.

### PR DESCRIPTION
**Pull request by GilmarPozzo**
- Create new Section
 - Get all Sections
 - Get Section
 - Update Section // not totally supported
 - Delete Section

### CODE REVIEW

**Comment by GilmarPozzo** [comment]
report todoist sections

File: config.json.dist [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "baseUri": "https://api.todoist.com/rest/v1",
     "credentials": {
       "owner": {
-        "token": ""
+        "token": "3c79150ab7e966c118b31207525ddf2e71939b32"
```
Comment (marcos-x86) [inline config.json.dist]: Please consider removing your personal token from config.json.dist file.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

File: config.json.dist (hunk 1 above)
Comment (GilmarPozzo) [reply config.json.dist]: Done!

File: config.json.dist [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+        "clientId": "",
+        "clientSecret": "",
+        "securityToken": "",
+        "userName": "",
+        "password": "",
+      }
+    }
+  }
+}
+=======
```
Comment (marcos-x86) [inline config.json.dist:68]: Please consider copying all the content from config.json.dist from develop branch to your branch in order to avoid conflicts.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

File: config.json.dist (hunk 2 above)
Comment (marcos-x86) [reply config.json.dist:68]: Also, please update your branch according to new changes in develop branch.

**Pull request closed without merging**